app/dashboard/views/alias_transfer.py

```diff
@@ -92,37 +104,35 @@ def alias_transfer_send_route(alias_id):
         )
         return redirect(url_for("dashboard.index"))
 
-    if alias.transfer_token:
-        alias_transfer_url = (
-            URL + "/dashboard/alias_transfer/receive" + f"?token={alias.transfer_token}"
-        )
-    else:
-        alias_transfer_url = None
+    alias_transfer_url = None
 
-    # generate a new transfer_token
     if request.method == "POST":
+        # generate a new transfer_token
         if request.form.get("form-name") == "create":
-            alias.transfer_token = str(uuid4())
+            transfer_token = f"{alias.id}.{secrets.token_urlsafe(32)}"
+            alias.transfer_token = hmac_alias_transfer_token(transfer_token)
+            alias.transfer_token_expiration = arrow.utcnow().shift(minutes=30)
```

Comment:
I think 30 minutes is a bit too low as the other person might not be available to receive the alias immediately. 24h seems more suitable to me.

Reply:
Updated to 24 hours.